9.secretstore/README.md

```diff
@@ -0,0 +1,144 @@
+# Secrets API
+
+This tutorial will help you write a node app to access secrets stored in a Secret Store using the Dapr Secrets API.
+
+
+
+## Prerequisites
+This sample requires you to have the following installed on your machine:
+- [Docker](https://docs.docker.com/)
+- [Node.js version 8 or greater](https://nodejs.org/en/)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- A Kubernetes cluster, such as [Minikube](https://github.com/dapr/docs/blob/master/getting-started/environment-setup.md#setup-cluster), [AKS](https://github.com/dapr/docs/blob/master/getting-started/environment-setup.md#setup-cluster) or [GKE](https://cloud.google.com/kubernetes-engine/)
```

Comment:
Can we use these links instead:

Minikube: https://github.com/dapr/docs/blob/master/getting-started/cluster/setup-minikube.md
Aks: https://github.com/dapr/docs/blob/master/getting-started/cluster/setup-aks.md

Reply:
Done. We need modify sample2 as well then because I got it from there

Reply:
Yes, [filed an issue](https://github.com/dapr/samples/issues/213).